npm-shrinkwrap.json

```diff
@@ -2,16 +2,6 @@
   "name": "paella-engage-ui",
```

Comment:
Yeah, you can, but it goes several levels deep into dependencies, so there are usually a lot of diffs, unless you've only change a shallow dependency that has no children.